engine/apps/api/serializers/alert_group.py

```diff
@@ -125,6 +125,7 @@ class AlertGroupListSerializer(
     PREFETCH_RELATED = [
         "dependent_alert_groups",
         "log_records__author",
+        "labels",
```

Comment:
This reduces the number of queries fetching labels for alert groups in one query instead of one per alert group.

Reply:
Just double-checking if it should be select relates, since alert_group - labels is a one to many relation

Reply:
In this case there are many labels per alert group, so we need to prefetch them (select related would be if there was a FK from alert group to label but here we have FK from label to alert group).